src/rss_cell_group.hpp

```diff
@@ -38,12 +38,17 @@ class rss_cell_group: public cell_group {
 
     void advance(epoch ep, time_type dt, const event_lane_subrange& events) override {
         for (auto& cell: cells_) {
-            auto t = std::max(cell.start_time, cell.t);
+            auto t_start = std::max(cell.start_time, cell.t);
             auto t_end = std::min(cell.stop_time, ep.tfinal);
 
+            std::size_t step = 0;
```

Comment:
To have full reproducable sequences, use `t = cell.start_time+cell.step*cell.period`, and increment `cell.step` (and set it to 0 when resetting the cell).

Reply:
Done